.gitignore

```diff
@@ -5,3 +5,4 @@ lib
 tmp
 *-error.log
 ./package-lock.json
+integrity.json
```

Comment:
Integrity file is never committed to the repository, but is published to NPM in order to do the integrity diffing.